src/ReactiveUI/ViewLocator.cs

```diff
@@ -93,31 +95,27 @@ public DefaultViewLocator(Func<string, string> viewModelToViewFunc = null)
         public IViewFor ResolveView<T>(T viewModel, string contract = null)
```

Comment:
Had a quick double check, and have made the only needed change (I think the docs included more functionality than the code did!)